columnflow/inference/__init__.py

```diff
@@ -264,6 +265,7 @@ def category_spec(
               when *config_data_datasets* is not defined, make of a fake data contribution.
             - *mc_stats*: Either *None* to disable MC stat uncertainties, or a float or tuple of
               floats to control the options of MC stat options.
+            - *fill_empty_bins*: When bins are empty, they are filled with this value.
```

Comment:
It would be nice to change the doc string so sphinx can automatically pick it up correctly. This shouldn't be too much work either I think, would you mind changing it?

Reply:
Fully agree, but it's actually quite a lot of work (the two files have 1900 lines of code with 40% being docs) that should perhaps be done in a separate PR to have *this*PR only adding the new logic.

I'd open the second PR in a bit.